### DISCUSSION TITLE
Clarify wording of branch protection setting

### DISCUSSION THREAD

**Discussion by mxdvl** [community/community#50256] [Repositories]
### Select Topic Area

Product Feedback

### Body

The use of negation creates unnecessary cognitive load on one of the branch protection setting checkboxes: “Do not allow bypassing the above settings”.

Aligning the language a positive action verb like other checkboxes on the page may improve accessibility for all users.

I believe “Allow bypassing the above settings” would be simpler, but the behaviour should be the opposite of the current implementation.

While this may be anecdotal, I personally have to double-check and re-read this setting everytime I encouter it. This feeling has been shared by some of my colleague, most recently @ioannakok.

> - **Require** a pull request before merging
> - **Require** approvals
> - **Dismiss** stale pull request approvals when new commits are pushed
> - **Require** review from Code Owners
> - **Require** approval of the most recent reviewable push
> - **Require** [status checks](https://docs.github.com/rest/commits/statuses) to pass before merging
> - **Require** branches to be up to date before merging
> - **Require** conversation resolution before merging
> - **Require** signed commits
> - **Require** deployments to succeed before merging
> - **Lock** branch
> - **_Do not allow_** bypassing the above settings 😖  
> - **Allow** force pushes
> - **Allow** deletions

---

I previously raised this as an issue in the docs repo, which was not the right forum: https://github.com/github/docs/issues/24417

**Comment by cehfisher** [comment]
Hi @mxdvl - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Reply by mxdvl** [reply]
This seems to me like an accessibility concern. Confusing language, especially when the vast majority of the world humans' first language is not English, hinders the access and confidence in using Github's UI…

**Reply by queenofcorgis** [reply]
Hey there!  👋 

Thanks for posting in the GitHub Community and the concern, @mxdvl.  The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
